pkg/server/login/locales.go

```diff
@@ -0,0 +1,56 @@
+package login
+
+import "golang.org/x/text/language"
+
+func getLocale(langBase string) map[string]string {
+	switch langBase {
+	case language.English.String():
```

Comment:
How do I know this works?

Reply:
Tested these methods with various  Accept-Language header locally and it was working. Not sure why?